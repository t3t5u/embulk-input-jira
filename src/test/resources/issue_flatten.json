{
    "simple": {
        "primitive": 1,
        "string": "string",
        "array": [],
        "object": {},
        "null": null
    },
    "twoLevels": {
        "1stLevel": {
            "2ndLevelPrimitive": 1,
            "2ndLevelString": "string",
            "2ndLevelArray": [],
            "2ndLevelObject": {},
            "2ndLevelNull": null
        }
    },
    "threeLevels": {
        "1stLevel": {
            "2ndLevel": {
                "3rdLevelPrimitive": 1,
                "3rdLevelString": "string",
                "3rdLevelArray": [],
                "3rdLevelObject": {},
                "3rdLevelNull": null
            }
        }
    }
}