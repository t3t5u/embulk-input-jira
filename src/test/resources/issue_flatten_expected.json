{
    "simple": {
        "primitive": 1,
        "string": "string",
        "array": "String value",
        "object": {},
        "null": null
    },
    "twoLevels": {
        "1stLevel.2ndLevelPrimitive": 1,
        "1stLevel.2ndLevelString": "string",
        "1stLevel.2ndLevelArray": "String value",
        "1stLevel.2ndLevelObject": {},
        "1stLevel.2ndLevelNull": null
    },
    "threeLevels": {
        "1stLevel.2ndLevel.3rdLevelPrimitive": 1,
        "1stLevel.2ndLevel.3rdLevelString": "string",
        "1stLevel.2ndLevel.3rdLevelArray": "String value",
        "1stLevel.2ndLevel.3rdLevelObject": "{}",
        "1stLevel.2ndLevel.3rdLevelNull": null
    }
}